less/jumbotron.less

```diff
@@ -9,6 +9,27 @@
   background-color: @jumbotron-bg;
   transition: background-color 0.2s ease-in;
 
+  &.bg-focus {
+    h1, .h1 {
+      color: @brand-navy;
+    }
+
+    .lead {
+      color: @brand-grey-plus-10;
+    }
+  }
```

Comment:
I think these are the default colours for these things anyway.  The only reason for having them is to overule the jumbotron styles, but over-ruling back to what they would have been anyway sucks.  That's why I was thinking we could move those text colours onto `.jumbotron-image` and `.hero-with-background` so we no longer have styles on jumbotron by default.